packages/clerk-js/src/ui/elements/Accordion.tsx

```diff
@@ -25,9 +25,15 @@ export const AccordionItem = (props: AccordionItemProps) => {
     }
   };
 
+  React.useEffect(() => {
+    setIsOpen(defaultOpen);
+  }, [defaultOpen]);
+
   React.useLayoutEffect(() => {
-    if (scrollOnOpen && isOpen && contentRef.current) {
-      contentRef.current.scrollIntoView({ behavior: 'smooth', block: 'center' });
+    if (scrollOnOpen && isOpen) {
+      window.requestAnimationFrame(() => {
```

Comment:
🔧 Since `requestAnimationFrame` is used in a `useLayoutEffect` we should return a function with a cancelAnimationFrame to be 100% correct. PTAL https://css-tricks.com/using-requestanimationframe-with-react-hooks/